.github/workflows/checks.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous integration
+on: [push]
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.5-alpine
+        ports:
+        - 5433:5432
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
+    - name: Checkout git repo
+      uses: actions/checkout@v2
+    - name: Create test databases
+      run: |
+        psql -U postgres -h localhost -p 5433 -c 'CREATE DATABASE lms_test'
+        psql -U postgres -h localhost -p 5433 -c 'CREATE DATABASE lms_functests'
+        psql -U postgres -h localhost -p 5433 -c 'CREATE DATABASE lms_bddtests'
+    - name: Build frontend assets
+      run: |
+        yarn install
+        yarn build
```

Comment:
yarn seems to be already installed

Reply:
@robertknight Could use your feedback on this: here I'm running `yarn install` and `yarn build` for the sake of the _backend_ functional tests, not the frontend stuff (these commands are being run in the backend VM not the frontend one). The backend functests crash with `assets.py` complaining about missing files if I don't do this. Is it really necessary to run a full `yarn` `install` and `build` here?